app/views/cart/show.html.erb

```diff
@@ -31,6 +31,8 @@
       <td>
         <center>
           <p><%=quantity%></p>
+		<p><%= button_to "+1", "/cart/#{item.id}/increment",method: :patch %></p> 
```

Comment:
I never knew you could do this!!! this is the quantity button for the items I take it? How do you stop it when the max quantity is reached?

Reply:
There is a method called `limit_reached?` inside the cart model that  looks like it is checking when the inventory of an item has been exhausted.